### PR TITLE
Add conformance test that verifies we can use runtime outdir with output glob

### LINKED ISSUE #109: new conformance test: $(runtime.outdir) for outputBinding glob
Found via https://dev.arvados.org/issues/17801

### PR DESCRIPTION
**Pull request by kinow**
Closes #109 

First time writing a conformance test for 1.2.1 :tada: 

After installing dependencies and learning how to use `run_tests.sh`, looked at the example in the issue and at existing tests to create this simple test.

Let me know if there are any mistakes, or good practices that I haven't used. I'm looking at the easy issues for now to learn how to write conformance tests.

Thanks!
Bruno

### CODE REVIEW

File: conformance_tests.yaml [hunk 1]
```diff
@@ -3501,3 +3501,33 @@
...
+          "class": "File",
+          "size": 0
+        }
+      ]
+    }
+  }
+  tool: tests/runtime-outdir.cwl
+  doc: |
+    Use of $(runtime.outdir) for outputBinding glob.
+  tags: [ required, command_line_tool, shell_command ]
```
Comment (mr-c) [inline conformance_tests.yaml:3533]: Can't have `required` if there is an optional feature used like `ShellCommandRequirement`. Can you adjust the test so it doesn't need `ShellCommandRequirement`?

**Review by mr-c — CHANGES_REQUESTED** [review]

File: conformance_tests.yaml (hunk 1 above)
Comment (kinow) [reply conformance_tests.yaml]: >Can't have required if there is an optional feature used like ShellCommandRequirement.

Noted! Thanks!

>Can you adjust the test so it doesn't need ShellCommandRequirement?

Tried, not sure if it's correct now, :crossed_fingers: 

Thanks Michael!

**Comment by kinow** [comment]
(rebased)

**Review by mr-c — APPROVED** [review]

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**